app/components/SearchResults.tsx

```diff
@@ -0,0 +1,90 @@
+import { Link } from "@remix-run/react"
+import React from "react"
+import type JSXNode from "~/types/JSXNode"
+import SanitizedTextSpan from "./SanitizedTextSpan"
+
+interface SearchResultsProps {
+  data: any
```

Comment:
Yeah so right now the type from the Loader is `Record<string, array[]>` from some backend code. But since we're changing the backend code anyway (to have multiple tabs and paginate each tab separately) this will have to probably change again.

I changed it to the type it currently is (`Record<string, any[]>`) which is something, but once I get the pagination stuff going I can make a better type